fastify.js

```diff
@@ -298,16 +285,27 @@ function build (options) {
     }
 
     req.id = genReqId(req)
-    handleTrustProxy(req)
-    req.hostname = req.hostname || req.headers['host']
+
+    var hostname = req.headers['host']
+    var ip = req.connection.remoteAddress
+    var ips
+
+    if (options.trustProxy) {
```

Comment:
Can you please make this a `const` in the outer scope?

Reply:
Done and there are references to `options.trustProxy` in getTrustProxyFn function also. Updated them accordingly. fe3b111